spec/models/user_spec.rb

```diff
@@ -53,9 +55,29 @@
     expect(user.contacts).to include(contact)
   end
 
+  it "should have no relationship to social accounts" do
+    user_with_no_social_accounts = FactoryBot.build(:user)
+    expect(user_with_no_social_accounts.social_accounts).to be_empty
+  end
+
+  it "can have many social accounts" do
+    user_with_social_accounts = FactoryBot.create(:user)
+    user_with_social_accounts.social_accounts.create(social_network: "GitHub", user_name: "SomeGitUserName")
+    user_with_social_accounts.social_accounts.create(social_network: "Telegram", user_name: "SomeTelegramUserName")
+
+    expect(user_with_social_accounts.social_accounts).not_to be_empty
+    expect(user_with_social_accounts.social_accounts[0].social_network).to eq("GitHub")
+    expect(user_with_social_accounts.social_accounts[1].social_network).to eq("Telegram")
+  end
+
+  it "adds user to contact request list of other user" do
```

Comment:
This test already exists below (in the proper scope). It is then also not necessary to move the `let` in a more general context